examples/cluster.yaml

```diff
@@ -62,6 +62,11 @@ spec:
   aws:
     region: "eu-central-1"
     az: "eu-central-1a"
+    elb:
```

Comment:
I think this file is stale, sorry about that I'll propose a PR for removing it. Could you please add the change to the example resource chart here https://github.com/giantswarm/aws-operator/blob/master/examples/local/aws-resource-lab-chart/templates/cluster.yaml ?

Reply:
Yep, will do